src/VisualStudio/Xaml/Impl/XamlVisualStudio.csproj

```diff
@@ -27,6 +27,10 @@
       <Project>{1EE8CAD3-55F9-4D91-96B2-084641DA9A6C}</Project>
       <Name>CodeAnalysis</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\CSharpCodeAnalysis.csproj">
```

Comment:
This would probably cause RPS regressions if we were to have any WPF tests using VB. I suspect the fix is elsewhere.